assets/lang/zh_CN.ini

```diff
@@ -1183,5 +1197,6 @@ Download = 下载
 New version of PPSSPP available = 新版本 PPSSPP 可用
 
 [Search]
-Find settings = 搜索设置
-Filter = 筛选
+Filtering settings by '%1' = Filtering settings by '%1'
+#Find settings = 搜索设置
+#Filter = 筛选
```

Comment:
Hm, these are used as long as your device has a keyboard.

-[Unknown]

Reply:
Ufff, nice catch.

I run the lang tools to sync missing line. Seems like it commended unused one too for some reason?

SDL build doesn't have the search function, anyway I'll have to fix this.